src/Sulu/Bundle/ContentBundle/Resources/translations/sulu/backend.de.xlf

```diff
@@ -707,6 +707,26 @@
                 <source>labels.error.content-unpublish-desc</source>
                 <target>Veröffentlichung der Seite konnte nicht widerrufen werden.</target>
             </trans-unit>
+            <trans-unit id="48" resname="content-navigation.security.texteditor">
+                <source>content-navigation.security.texteditor</source>
+                <target>Text-Editor</target>
+            </trans-unit>
+            <trans-unit id="49" resname="security.roles.ckeditor.title">
+                <source>security.roles.ckeditor.title</source>
+                <target>Einstellungen für Text-Editor</target>
+            </trans-unit>
+            <trans-unit id="50" resname="security.roles.ckeditor.type">
+                <source>security.roles.ckeditor.type</source>
+                <target>Sektion</target>
+            </trans-unit>
+            <trans-unit id="51" resname="security.roles.ckeditor.horizontal">
+                <source>security.roles.ckeditor.horizontal</source>
+                <target>Funktion</target>
+            </trans-unit>
+            <trans-unit id="52" resname="security.roles.ckeditor.info">
+                <source>security.roles.ckeditor.info</source>
+                <target>Die Konfiguration gilt für jeden Text-Editor im System, der von dieser Rolle verwendet wird.</target>
+            </trans-unit>
```

Comment:
We've agreed to namespace the translations with the bundle name, so these translations should look like `sulu-content.ckeditor.roles-info`and so on.